docs/source/debug.rst

```diff
@@ -178,3 +178,15 @@ Execuitng the EBREAK instruction when the core is in Debug Mode shall result in
 
 - The core remains in Debug Mode and execution jumps back to the beginning of the debug code located at ``dm_halt_addr_i``
 - none of the CSRs are modified
+
+
+.. _interrupts_single-stepping:
+
+Interrupts during Single-Step Behavior
+--------------------------------------
+
+The CV32E40P CPU is compliant with the 0.13.2 RISC-V Debug spec.
```

Comment:
I think this is a bit too strong. I would say that the 40P is not compliant to the intended interpretation of the 0.13.2 specification, but that this intended behavior was only clarified in version 1.0.0.

Reply:
agree and fixed

Reply:
let's wait for @strichmo before merging in